CHANGELOG.md

```diff
@@ -8,6 +8,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Changed (Breaking)
+
+- Apply underscore pattern to modules (#993)
+  - AccessControlComponent
+    - `_change_role_admin` function renamed to `change_role_admin`
+  - PausableComponent
+    - `_pause` function renamed to `pause`
+    - `_unpause` function renamed to `unpause`
+  - ERC20Component:
+    - `_mint` function renamed to `mint`
+    - `_burn` function renamed to `burn`
+  - ERC721Component:
+    - `_safe_transfer` function renamed to `safe_transfer`
+    - `_safe_mint` function renamed to `safe_mint`
+    - `_burn` function renamed to `burn`
```

Comment:
Should we change `_mint` to `mint` since we're doing that with erc20?

Reply:
Similar to _transfer in ERC721, I think we should keep _mint underscored because we don't want users to use them, those are private helpers for the "safe" versions that are not underscored (safe_mint and safe_transfer)

Reply:
> Similar to _transfer in ERC721, I think we should keep _mint underscored because we don't want users to use them

My impression was that we were taking an agnostic stance regarding x and safe_x because there's some risk in both approaches. They'll still be accessible and this is more of an organizational change, so it's not the end of the world if you prefer it this way. I'm just not a fan of blanket-favoring the safe variants

Reply:
mmm, that's a very good point. Agree.